api/src/test/java/org/openmrs/OrderTest.java

```diff
@@ -511,6 +514,32 @@ public void isDiscontinued_shouldReturnTrueIfCheckDateIsAfterBothDateStoppedAuto
 	}
 	
 	/**
+	 * @see Order#isDiscontinued(Date)
+	 * @verifies return true if the order is scheduled for the future and activated on check date
+	 *           but the check date is after date stopped
+	 */
+	@Test
+	public void isDiscontinued_shouldReturnTrueIfTheOrderIsScheduledForTheFutureAndActivatedOnCheckDateButTheCheckDateIsAfterDateStopped()
+	        throws Exception {
+		// tests the case when a scheduled order is revised:
+		// in that case its original order is stopped.
+		// the stopped date of the original order is set to a moment before the activated date of the revised order.
+		// the order here represents such an original order
+		Order order = new Order();
+		order.setUrgency(Urgency.ON_SCHEDULED_DATE);
+		Date scheduledDateInFuture = DateUtils.addMonths(new Date(), 2);
```

Comment:
Can you add a local variable and assign new Date() to it? So that you re-use it instead of creating multiple instances